### ISSUE TITLE
Need a standard debug logging class that isn't console.writeline

### ISSUE THREAD

**Issue by noita-player** [shalzuth/LostArkLogger#63]
Problems:

- [ ] Needs to support: output to stdout, out to file, and both at the same time. Electronbridge might need stdout. #62 
- [ ] Different debug levels, ideally, such as error vs info - so we can have an info-log that is very small output and always there, then enable the debug flag and have it dump debug-level messages too that might be more spew-ey
- [ ] Can we get line numbers for exceptions in release builds? Maybe if the logger is passed an exception object? Do we need to ship full PDBs?

**Comment by shalzuth** [comment]
I agree, but having a dependency on not using stdout isn't acceptable.
PDB is required for line numbers, i can include that in the artifacts (but user will need to download it in addition to the exe)

**Comment by shalzuth** [comment]
Logger class could prevent stdout from normal log calls for electronbridge as a temporary means
